packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx

```diff
@@ -23,12 +23,15 @@ export const TxRecipientCard: TxRecipientCardComponent = ({
   recipient,
   isReceiver,
 }) => {
+  const { accounts } = useAccounts();
   const address = recipient?.address || '';
   const isValidAddress = isB256(address) || isBech32(address);
   const fuelAddress = isValidAddress
     ? Address.fromString(address).toString()
     : '';
   const isContract = recipient?.type === AddressType.contract;
+  const name =
+    accounts?.find((a) => a.address === fuelAddress)?.name || 'unknown';
```

Comment:
I think `unknown` it's not looking good on this context

<img width="352" alt="image" src="https://user-images.githubusercontent.com/8636507/229014232-d2cc7fcc-86af-4825-ab90-8ef1fb2fd623.png">
<img width="354" alt="image" src="https://user-images.githubusercontent.com/8636507/229014750-597203c1-feea-4787-a7a1-1a96a4417056.png">

We should move address to first position (avoid holes in layout), and put the account name below the address. also reduce highlight of account name, reducing font-size and making spacing smaller

Reply:
@LuizAsFight - Switching the positions of Account `name` and `address` breaks design consistency. I'd probably recommend adding a warning icon ⚠️ beside the `unknown` to emphasise the user to double check the address they're transferring funds to, thoughts?
cc: @luizstacio

Reply:
Let's keep the consistency and the name on the top, and use unknow when we don't have it. I think is no need for the warning icon at this point.

Reply:
`unknown` is weird and should be removed in my opinion but I won't block the PR because of this, you guys decide it. I never seem `unknown` when approving transaction in metamask for example.

the only data _**needed**_ to a transfer is the address
account name is extra data, to increase level of information provided. but it's not needed.
regarding UX highlighting order, main data should be shown first, extra data after.

this screen will **almost always** show "unknown", which brings unsafe thoughts in user's mind.
it will show other thing only when the account is registered in the wallet, which means the user owns it and have the keys

Reply:
cc @luizstacio

Reply:
then when it's not unknown, we show...